runhouse/servers/obj_store.py

```diff
@@ -1514,28 +1521,193 @@ def status_local(self):
                     else (py_module + "." + cls.__qualname__)
                 )
                 if isinstance(v, runhouse.Resource):
-                    resources_in_env_modified.append(
+                    objects_in_env_modified.append(
                         {"name": k, "resource_type": cls_name}
                     )
                 else:
-                    resources_in_env_modified.append(
+                    objects_in_env_modified.append(
                         {"name": k, "resource_type": cls_name}
                     )
-            return resources_in_env_modified
-        else:
-            return []
+
+        if env_name:
+            list_env_actor = (
+                subprocess.check_output(
+                    [
+                        "ray",
+                        "list",
+                        "actors",
+                        "-f",
+                        "state=ALIVE",
+                        "-f",
+                        f"name={env_name}",
+                    ]
+                )
+                .decode("utf-8")
+                .split("\n")
+            )
+
+            total_memory = psutil.virtual_memory().total
+
+            # The 'real' info about the env_servlet actors starts at index 9 of the list which represent the cli output.
+            # (list_env_actors)
+            # The info about the env_servlets is stored in a table. Each record in the table describes one env_servlet
+            # in the following format:
+            # record_id(0) ACTOR_ID(1) CLASS_NAME(2) STATE(3) JOB_ID(4) NAME(5) NODE_ID(6) PID(7) RAY_NAMESPACE(8)
+            env_actor_info = list_env_actor[9].split()
+            env_pid = env_actor_info[7]
+            env_node_info = (
+                subprocess.check_output(["ray", "get", "nodes", env_actor_info[6]])
+                .decode("utf-8")
+                .split("\n")
+            )
+            node_ip = env_node_info[4].split(":")[1].strip()
+            node_name = "head"
+            env_servlet_pid = int(env_actor_info[7])
+            if not cluster_config.get("resource_subtype") == "Cluster":
+                stable_internal_external_ips = cluster_config.get(
+                    "stable_internal_external_ips"
+                )
+                for ips_set in stable_internal_external_ips:
+                    if ips_set[0] == node_ip:
+                        # head ip == cluster address == cluster.ips[0]
+                        if ips_set[1] == cluster_config.get("ips")[0]:
+                            node_name = "head"
+                        else:
+                            node_name = f"worker{stable_internal_external_ips.index(ips_set) + 1}"
```

Comment:
are we showing the IP of the nodes anywhere? otherwise if we just take the index and there's multiple worker nodes having the number here won't be so informative

Reply:
Added to the node name the ip itself, will print `head (<ip>)` or `workerX (<ip>)`